3-calc-vars/src/expr.rs

```diff
@@ -13,16 +16,23 @@ pub enum BinOp {
     Div,
 }
 
+impl BinOp {
+    pub fn to_fn(self) -> fn(f64, f64) -> f64 {
```

Comment:
`BinOp::to_fn` doesn't have to be public to be accessible to `Expr::eval` - visibility is limited by module boundaries, `impl` blocks merely group methods of a given type.